app/controllers/settings/hostings_controller.rb

```diff
@@ -17,6 +17,17 @@ def update
     end
   end
 
+  def send_test_email
+    current_user_email = Current.user.email
+    begin
+      NotificationMailer.test_email(current_user_email).deliver_now
```

Comment:
For consistency with the [Rails convention](https://guides.rubyonrails.org/action_mailer_basics.html#mailers-are-similar-to-controllers), can we invoke this as `NotificationMailer.with(user: Current.user).test_email` and then obtain the email later with params[:user].email?